index.html

```diff
@@ -1,284 +1,309 @@
+  <title>My Portfolio</title>
+</head>
+
+<body>
+  <div>
+    <header>
+      <a href="homepage.html">
+        <img src="./assets/My Logo.svg" class="logo" /></a>
       <nav class="navbar">
         <img class="menu" src="./assets/Side.svg" alt="menu" />
+
+        <ul class="desktop-menu-bar">
+          <li class="nav"><a href="#Portfolio">Portfolio</a></li>
+          <li class="nav"><a href="#aboutme">About</a></li>
+          <li class="nav"><a href="#contact-me">Contact</a></li>
+        </ul>
+
       </nav>
     </header>
-      <section class="my-section">
-        <div>
-          <h1 class="intro">I'm Walid <br />Glad to see <br />You!</h1>
-        </div>
-        <div>
-          <p class="about-me">
-            I’m a software developer! I can help you <br />
-            build a product , feature or website Look <br />
-            through some of my work and <br />
-            experience! If you like what you see and <br />
-            have a project you need coded, don’t <br />
-            hestiate to contact me.
-          </p>
-        </div>
-        <div class="contact-me">
-          <h2 class="connect">let's connect</h2>
-        </div>
-        <div>
-          <ul class="buttons">
-          <li><a href=""><img src="./assets/Github.svg"  class="contact-btn" alt="Github" /></li></a>
-          <li><a href=""><img src="./assets/Twitter.svg" class="contact-btn"  alt="Twitter" /></li></a>
-          <li><a href=""><img src="./assets/medium.svg"  class="contact-btn" alt="Medium" /></li></a>
-          <li><a href=""><img src="./assets/Vector.svg"  class="contact-btn" alt="" /></li></a>
+    <section class="my-section">
+      <div>
+        <h1 class="intro">I'm Walid <br />Glad to see You!</h1>
+      </div>
+      <div>
+        <p class="about-me">
+          I’m a software developer! I can help you 
+          build a product , feature or website Look 
+          through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don’t 
+          hestiate to contact me.
+        </p>
+      </div>
+      <div class="contact-me">
+        <h2 class="connect">let's connect</h2>
+      </div>
+      <div>
+        <ul class="buttons">
+          <li><a href=""><img src="./assets/Github.svg" class="contact-btn" alt="Github" /></li></a>
+          <li><a href=""><img src="./assets/Twitter.svg" class="contact-btn" alt="Twitter" /></li></a>
+          <li><a href=""><img src="./assets/medium.svg" class="contact-btn" alt="Medium" /></li></a>
+          <li><a href=""><img src="./assets/Vector.svg" class="contact-btn" alt="" /></li></a>
         </ul>
-        </div>
-      </section>
+      </div>
+    </section>
 
-      <section class="card">
+    <section class="card">
       <div class="grid-container">
         <div class="first-card">
           <img class="card-img" src="/assets/First Snapshoot Portfolio.svg" alt="Tonic">
```

Comment:
- [ ] Good job implementing the desktop version :clap: , I think you missed a `.` here due to which the image for this card is not rendering properly please see :point_down:  please change it to the correct path like :point_right:  `src=./assets/First Snapshoot Portfolio.svg`
![image](https://user-images.githubusercontent.com/97521557/213234133-c32e996e-dbf5-4122-8b0f-af2998947b36.png)